### DISCUSSION TITLE
[Unintended Feature or Bug] Table of content with latex

### DISCUSSION THREAD

**Discussion by SPyofgame** [facelessuser/pymdown-extensions#1994] [Q&A] [answered]
Suppose I want to write an editorial to describe certain algorithms like this:

![image](https://user-images.githubusercontent.com/62207095/226192298-a421032c-d6af-4a84-9942-2610e5cefe67.png)

It did render like this, which is expected:

![image](https://user-images.githubusercontent.com/62207095/226192314-ac7bb091-86d9-4ff7-8723-a2ef0319bde8.png)

But in the table of content, it shows this:

![image](https://user-images.githubusercontent.com/62207095/226192334-2377ba66-a399-4572-ad15-884d7dc97c59.png)

**Comment by facelessuser** [comment, marked as answer]
Pictures are nice, but in the future, also post copy/pastable examples as well so I don't have to transcribe them from an image.

With all of that said, this is a documentation system issue, not an issue with these plugins. You could force the information into the toc:

```
## Bruteforces $O(4^n)$ {data-toc-label='<span class=arithmatex>\\(O(4^n)\\)</span>'}
```

But the documentation system still strips the HTML tag `<span>` out. That is needed, along with the class to get it translated. I guess if you adjust the MathJax config, you may be able to get it to target navigation link items, but you'd have to look into that.

**Reply by SPyofgame** [reply]
As an Insider, should I also report this to [squidfunk](https://github.com/squidfunk)'s pymdown too ?

**Reply by SPyofgame** [reply]
Seems like the side panel table of content works just fine, but not [TOC]

**Reply by facelessuser** [reply]
I adjusted my personal config to include navigation elements:

```
if (!('mathjaxConfig' in window)) {
  window.MathJax = {
    tex: {
      inlineMath: [["\\(", "\\)"]],
      displayMath: [["\\[", "\\]"]],
      processEscapes: true,
      processEnvironments: true,
      tagSide: "right",
      tagIndent: ".8em",
      multlineWidth: "85%",
      tags: "ams"
    },
    options: {
      ignoreHtmlClass: ".*",
      processHtmlClass: "arithmatex|md-nav__item"
    }
  }
}
```

This allowed it to work:

<img width="1007" alt="Screenshot 2023-03-19 at 10 17 31 PM" src="https://user-images.githubusercontent.com/1055125/226246176-a1053cf3-39be-44ff-913d-fce24d07e597.png">

You would have to do the same. Squidfunk maintains a MkDocs theme. I maintain a collection of Python Markdown extensions that Squidfunk's MkDocs theme relies on for certain features.

**Reply by SPyofgame** [reply]
It is nice to see that, thank you for your dedicated work❤️
